source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -3130,18 +3204,6 @@ void Graph::addNodePopup(bool cursor)
     }
     if (ImGui::BeginPopup("add node"))
     {
-
-        // check if there is a document
-        if (_graphDoc == nullptr)
```

Comment:
This late creation is no longer necessary as a document is always created on "new document".